frontend/src/Components/Link/index.tsx

```diff
@@ -43,6 +43,9 @@ const Link = () => {
       // 'payment_initiation' products do not require the public_token to be exchanged for an access_token.
       if (isPaymentInitiation) {
         dispatch({ type: "SET_STATE", state: { isItemAccess: false } });
+      } else if (isCraProductsExclusively){
+        // When only CRA products are enabled, only user_token is needed. access_token/public_token exchange is not needed.
+        dispatch({ type: "SET_STATE", state: { isItemAccess: true } });
```

Comment:
Why are we using a variable called `isItemAccess` when we're actually using a user token? I'm confused by what this is intended to mean.

Reply:
Ya, good point. I'm gonna set this to false here and add some `userToken` related logic elsewhere where`isItemAccess` is referenced.